### ISSUE TITLE
Wrong redirect on log out

### ISSUE THREAD

**Issue by Baukereg** [simplabs/ember-simple-auth#513]
I have my app running on a directory on my domain, like `http://mydomain.com/myapp/`, and the authenticationRoute set to index.

When I log out, ember-simple-auth redirects to `http://mydomain.com/`. I couldn't find any base url setting in the docs. Is there a way to fix this?

**Comment by amedrz** [comment]
I asume you're using ember-cli, have you set [applicationRootUrl](https://github.com/simplabs/ember-simple-auth/blob/master/packages/ember-simple-auth/lib/simple-auth/configuration.js#L152) in your environment.js?

**Comment by marcoow** [comment]
`applicationRootUrl` is auto detected from the [router's `rootURL`](http://emberjs.com/api/classes/Ember.Router.html#property_rootURL).

**Comment by marcoow** [comment]
This will be ok once #522 gets merged.

**Comment by knownasilya** [comment]
I can't wait! This plagues me lol

**Comment by knownasilya** [comment]
FYI

You can add the following to your ApplicationRoute, while we wait for 1.0.

```js
actions: {
  sessionInvalidationSucceeded() {
    if (!Ember.testing) {
      window.location.replace('/dashboard');
    }
  }
}
```

**Comment by marcoow** [comment]
will be fixed in #522
